.goreleaser.yml

```diff
@@ -195,7 +195,8 @@ brews:
         Logs:   #{var}/log/swarm-bee/bee.log
         Config: #{etc}/swarm-bee/bee.yaml
 
-        Bee has SWAP enabled and by default is using slock.it goerli ethereum endpoint.
+        Bee has SWAP enabled and by default is using ws://localhost:8546 ethereum endpoint.
+        If needed obtain Infura account and set it.
```

Comment:
I liked your comment in `packaging/docker/README.md` (line 15).

Reply:
I think we need to get rid of the SWAP construct from the CLI. Also, there is the storage incentives enabled by default and the backend is shared between them so we cannot and probably should not affiliate the ethereum backend just with a swap context

Reply:
@acud what wording to use if not SWAP?